packages/ledger-icp/src/ledger.canister.ts

```diff
@@ -29,7 +24,7 @@ import type {
   TransferRequest,
 } from "./types/ledger_converters";
 import { paramToAccountIdentifier } from "./utils/params.utils";
-import { importNnsProto, queryCall, updateCall } from "./utils/proto.utils";
+import { queryCall, updateCall } from "./utils/proto.utils";
```

Comment:
Will these dependencies be removed later?

Reply:
yes!